dicom/matRad_importDicomSteeringParticles.m

```diff
@@ -36,6 +36,10 @@
 %% load plan file
 % load machine data
 
+env = matRad_getEnvironment();
+matRad_checkEnvDicomRequirements(env);
+isOctave = strcmp(env,'OCTAVE');
```

Comment:
This can also be directly obtained through matRad_cfg (from the MatRad_Config). Maybe switch to this in all scripts.

Reply:
Resolved for all scripts